packages/react-components/styles/fonts.v2.tsx

```diff
@@ -67,6 +67,12 @@ const fontStyles = StyleSheet.create({
   center: {
     textAlign: 'center',
   },
+  mediumNumber: {
```

Comment:
This is not in the Design Kit, but we used more than twice on Gold Screen - I figure we could use it the future. Cc @coreycelo